packages/syft/src/syft/service/action/action_object.py

```diff
@@ -579,6 +713,19 @@ def _syft_try_to_save_to_store(self, obj) -> None:
         # relative
         from ...client.api import APIRegistry
 
+        if obj.syft_node_location is None:
+            obj.syft_node_location = obj.syft_node_uid
```

Comment:
There is not much difference, syft_node_location is set everytime an object is passed from service_method decorator. While syft_node_uid is used to store the id of the node when its saved. Since both of them were developed in parallel, there is a slight redundancy in their use. I agree we should combine them, but would prefer to do that in a seperate PR.